src/Giveaway.js

```diff
@@ -160,7 +160,7 @@ class Giveaway extends EventEmitter {
      * @type {Discord.PermissionResolvable[]}
      */
     get exemptPermissions() {
-        return this.options.exemptPermissions || this.manager.options.default.exemptPermissions;
+        return this.options.exemptPermissions.length ? this.options.exemptPermissions : this.manager.options.default.exemptPermissions;
```

Comment:
I don't see the point of adding a `.length` here. If it is an empty array, it should be a valid value right? If the point of this `.length` is to check whether the value is an array, there is a proper way to do it using `Array.isArray()`.

Btw if the value is `undefined`, it will throw `cannot read property length of undefined` right?

Reply:
In the `data` getter I made `exemptMembers` to an empty array per default because the `manager.default` one is also an empty array per default. 
And that's why the `.length` is in place to check if  the array has any values because as said per default it would now be an empty array and `||` would not work with that.
Oh, but now I get it, before a restart would happen it would be still be `undefined` if it's not set because the DB data gets fetched only on startup.
So just add a `Array.isArray()` then right?

Reply:
Yes, check if it is an array and if it is not, use the default manager option

Reply:
Or would it maybe be better to move what I added in the `data` getter to the `.start` fuction so that it would be a emtpy array per default on giveaway creation?

Reply:
Hmm yes we can, when instantiating the Giveaway object (https://github.com/Androz2091/discord-giveaways/blob/develop/src/Manager.js#L200)

Reply:
Changed